### ISSUE TITLE
Please adjust 3.1.1 - the definitions of core and supplemental, so WCAG 3 is more equitable.

### ISSUE THREAD

**Issue by lseeman** [w3c/wcag3#653]
#Please adjust 3.1.1 - the definitions of core and supplemental, so WCAG 3 is more equitable.
## Issue description:
We are concerned that core requirements, as they are currently defined, are inequitable between disability groups. This is both theoretical and in practice. It’s biased theoretically as both the definitions of core and supplemental seem to be biased towards traditional AT users, such as screen readers. And it is biased in practice as we have seen many requirements that are essential to cognitive users, but because  we do not always use traditional AT,  these requirements are downgraded to supplemental. 

WCAG 3 is trying to equitably meet the needs of more user groups, such as cognitive disabilities and low vision, but these definitions work against that goal. 

##Core requirements:
 We propose that core requirements are items that are essential to the user. We define requirements as essential to the user if users can not independently  use the content without this requirement because of their disability.  

If this is included in the fourth bullet about “necessary support,” then please specify that this is part of necessary support. For example:

necessary support is available such as captions, text alternatives, and expanded forms of abbreviations and any requirement that are essential to the user. IE they can't independently  use the content without this requirement because of their disability. 
If making core requirements focus on essential to the user, means that there are too many requirements as core, then we suggest that other bullets in the definition are removed. In any case, really essential support for any group and all groups is included in this one line of essential to the user.  

##Supplemental requirements:
 Items that are important to the user experience, but are not essential due to a disability can be supplemental. This could be a bad user experience but you can still use the content independently.  

The way WCAG 3’s definition of supplemental requirement is currently worded seems very subjective. For example, requirements that are currently more difficult or expensive to implement — more expensive than what? Such as adding audio descriptions to an hour lecture? Audio descriptions are currently a core requirement and are expensive and time-consuming.

The current definition of supplemental could be used to deprioritize some user groups. This would  include downgrading requirements that are essential to these user groups independently  using content. 

##More information 
 For reference, WCAG 3 currently defines three types of provisions in [Section 3.1.1](https://w3c.github.io/wcag3/guidelines/#types-of-provision) as follows (so you do not have to find it):

Core Requirements — Requirements that must be met in order to conform. These include requirements that ensure:
content is detectable by user agents and AT,
content is conveyed to multiple senses and can be used by multiple inputs,
content does not cause immediate, physical harm, and
necessary support is available such as captions, text alternatives, and expanded forms of abbreviations.

Supplemental Requirements — Requirements that build on the core set. These include:
quality checks,
stricter or higher levels than those in a comparable core requirement,
requirements that are currently more difficult or expensive to implement, and
requirements that may apply in certain situations.

Assertions — Documented statements you can make about specific accessibility practices your organization follows to improve the accessibility of your content or product.

With thanks

The COGA task force

**Comment by GreggVan** [comment]
Lisa et al - thanks for all you do and contribute to advance cognitive, language, and learning disabilities.  

However, I  think it would be useful to stop calling this an equity issue.  There are more cognitive, language, and learning disabilities specific requirements than there are blindness specific provisions.     The problem is not equity - it is coverage.   There is much more variation needed in  cognitive, language, and learning disabilities.   And cognitive, language, and learning disabilities  does not **currently** have and use AT like blindness.    But there are very many blind people who cannot use screen readers and are more out in the cold than people with cognitive, language, and learning disabilities.       We currently only cover what I  would estimate as maybe 60% of people with disabilities - and that includes many type, degree, and combination of disability.   Cognitive, language, and learning disabilities is a large part of this -- but constantly claiming that the group is not being equitable I think is unfair to the group.   The working group - all the way back to WCAG 2.0 spent more time on cognitive, language, and learning disabilities  than on any other disability group (and I  think all the other disability groups combined).  The fact that cognitive, language, and learning disabilities is difficult and it has been and is hard to create objective requirements that can be applied broadly is the big culprit.  When we have identified things that could be included there has not been any problem in getting them adopted.  

Lets focus on finding qualified items.   And if there are any that seem qualified but are don't seem to be getting considered in one's opinion --  name that specific item (or items) and bring it to discussion.   General comments of we don't have everything we need therefore people are being inequitable - is not helpful I don't think.   

I can say personally that I  have spent at least twice as much time trying to find cognitive, language, and learning disabilities  provision that can work than all of the other provisions combined.   And I think that is a vast underestimate.   It is just hard.  Very hard.   

One of the reasons I  joined WCAG3 work was specifically because I  thought we needed to get more cognitive, language, and learning disabilities  in front of authors.   Either by identifying new valid requirements or by structuring it such that the parts of incredibly rich guidance in the cognitive access note that can't be a requirement  can be included in such a way that all authors see that guidance along with the requirements. 

It is also why I am championing the Info-Bot + IUIG approach that focuses on Individual User Interface Generators (IUIGs) that are tuned to the individual and completely sidesteps any need for testability. 

But at any rate - lets stop talking about the working group being inequitable or implying the same by saying its output is inequitable and focus instead on advancing things that can be required -- AND focusing on getting the great guidance that can't be required incorporated so it is seen alongside the requirements.

**Comment by jmcsorle** [comment]
I think it's important to allow people within this community to express their opinions about bias when they believe it exists and it's inappropriate to try to silence those opinions simply because you disagree with them. Lisa does not say "the working group is being inequitable." As you point out, WCAG does not cover the needs of all people with disabilities and calling it coverage instead of equity does not change that fact.

This should be a safe place for anyone to point out where they believe we need to do better, even when the issues they are highlighting are complex.

Yes, cognitive accessibility issues are complex, but it is both Lisa's right and her duty as the chair of COGA to point out where she believe bias / lack of coverage / lack of equity exists.

Given her own lived experience with disability and her decades of work with standards, it is disappointing to see a comment that appears to summarily dismiss her perspective.

**Comment by GreggVan** [comment]
@jmcsorle 
you wrote 
>I think it's important to allow people within this community to express their opinions about bias when they believe it exists and it's inappropriate to try to silence those opinions simply because you disagree with them.

1) Actually we have been repeatedly asked by the Chairs to stick to the topics and not talk about people.   Talking about bias - is talking about people and attributing bad intentions or at least bad behavior (intentional or not) to them.   I was only asking that we stick to the topics and not criticize people or accuse them of things like bias.    Which is outside of our rules of participation. 

2) I don't disagree with Lisa' mission. It is the only reason I am working on WCAG3 in retirement -- because I  think we can find ways to get more cognitive guidance in if we structure it differently.   Maybe not more requirements -- we have been working for decades on that - and that is really hard.   But we may find some more.  But more important I  thinks is getting good guidance - like that which the COGA NOTE is full of - into WCAG3 in ways that it will be seen by authors. 

Casting blame -- doesn't help.   Finding solutions - does.   And I truly think that the vast majority - if not all - of the working group is trying their best to get as much cognitive, language, and learning disability related content as they can into WCAG.

**Comment by lseeman** [comment]
Hi Gregg.
what can I say. Recollections may differ.

I am reminding you of my request to not bring up history.  We had very different experience in the early years in WCAG and it is counter productive and upsetting to discuss them.

For us the  guidelines needs to be about making content accessible for people with disabilities. This issue is about reframing section [Section 3.1.1](https://w3c.github.io/wcag3/guidelines/#types-of-provision)  so that it focuses on the importance to the end user, equally, regardless of disability. 

I understand that you think it is more important to be easy for legislators to incorporate wcag then for the guidelines to provided equal accommodation.  We disagree, although I think we both agree that some balance is necessary. As it stands 3.1.1 (the section on core and supplemental) is not balanced and not inclusive enough of cognitive disabilites.  At least from our perspective. We saw this this from reviewing the provisions and then wondered why why so many important provisions for us tended to be supplemental. We found this section and saw the the problem. Hence this issue.  We will also ask for some individual provisions to be change as we complete our review.

**Comment by GreggVan** [comment]
Hi Lisa
I agree that we should not talk about WCAG history.    Doesn't advance what we are doing.    Will not mention history if it is not mentioned.   Hard to not respond when only one version is presented   -- but as long as we all just look forward that is the only direction that will change things.

+1

**Comment by alastc** [comment]
Hi Lisa,

Sticking to the original topic, you wrote:
> It’s biased theoretically as both the definitions of core and supplemental seem to be biased towards traditional AT users, such as screen readers.

I think that might be correct in practice but not in theory: Theoretically, cognitive users could be using AT, and would benefit from doing so.

For example, text-to-speech is something I would assume is beneficial for a wide variety of users. Also, anyone using AI agents would benefit from the AT-oriented requirements (as [they use the accessibility tree](https://developer.chrome.com/docs/lighthouse/agentic-browsing/scoring#agent-centric_accessibility)).

In practice, the AT for people with cognitive issues is more varied, less known, and less available. So I understand your point. 

> We propose that core requirements are items that are essential to the user. 

Isn't that all the requirements? They are all there because they are essential to someone in some scenario. If they weren't, they would be a best-practice / recommendation.

You suggested:
> if users can not independently use the content without this requirement because of their disability.

Like all of these, it's a squishy & subjective thing, there will always be scenarios where someone can't use something. Are there examples in the [current draft](https://w3c.github.io/wcag3/guidelines/) that are supplemental that you think should be core? And would be adjusted with that definition?
